sdk/interchain/signature-aggregator.go

```diff
@@ -153,7 +153,11 @@ func NewSignatureAggregator(
 			logging.JSON.ConsoleEncoder(),
 		),
 	)
-	return initSignatureAggregator(peerNetwork, logger, registerer, subnetID, quorumPercentage)
+	etnaTime := time.Unix(0, 0)
+	if t, ok := constants.EtnaActivationTime[network.ID]; ok {
+		etnaTime = t
+	}
```

Comment:
```suggestion
	etnaTime := constants.EtnaActivationTime[network.ID]
```

This will use a default (i.e. zero) `time.Time` instance if the key is not in the map

Reply:
totally agree. fixed